Makefile

```diff
@@ -1,10 +1,37 @@
 .PHONY: build_macos
 build_macos:
 	@echo "Building Macos..."
-	gogio -icon=./assets/appicon.png -target=macos -arch=amd64 -o ./dist/amd64/Chapar.app .
-	gogio -icon=./assets/appicon.png -target=macos -arch=arm64 -o ./dist/arm64/Chapar.app .
+	gogio -icon=./build/appicon.png -target=macos -arch=amd64 -o ./dist/amd64/Chapar.app .
+	gogio -icon=./build/appicon.png -target=macos -arch=arm64 -o ./dist/arm64/Chapar.app .
 	codesign --force --deep --sign - ./dist/amd64/Chapar.app
 	codesign --force --deep --sign - ./dist/arm64/Chapar.app
+	tar -cJf ./dist/Chapar_macos_amd64.tar.xz --directory=./dist/amd64 Chapar.app
+	tar -cJf ./dist/Chapar_macos_arm64.tar.xz --directory=./dist/arm64 Chapar.app
+	rm -rf ./dist/amd64
+	rm -rf ./dist/arm64
+
+.PHONY: build_windows
+build_windows:
+	@echo "Building Windows..."
+	gogio -icon=build\appicon.png -buildmode=archive -target=windows -arch=amd64 -o dist\amd64\Chapar.exe .
+	gogio -icon=build\appicon.png -buildmode=archive -target=windows -arch=386 -o dist\i386\Chapar.exe .
+	rm *.syso
+	zip -r -j dist\Chapar_windows_amd64.zip dist\amd64\Chapar.exe
+	zip -r -j dist\Chapar_windows_i386.zip dist\i386\Chapar.exe
+	rm -rf dist\amd64
+	rm -rf dist\i386
+
+.PHONY: build_linux
+build_linux:
+	@echo "Building Linux amd64..."
+	CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build -o ./dist/amd64/chapar .
```

Comment:
I'm surprised to do not find this step of macOS and Windows

Reply:
They use gogio and the tools will do same thing behind the scenes